lib/Runtime/Types/DictionaryTypeHandler.cpp

```diff
@@ -2460,7 +2538,8 @@ namespace Js
     }
 
     template <typename T>
-    void DictionaryTypeHandlerBase<T>::InvalidateFixedField(DynamicObject* instance, PropertyId propertyId, DictionaryPropertyDescriptor<T>* descriptor)
+    template <typename TPropertyKey>
```

Comment:
>typename TPropertyKey [](start = 14, length = 21)

Dumb question. You have renamed template variable T to TPropertyKey. How does it work for DictionaryPropertyDescriptor<T>* descriptor in the signature?